### ISSUE TITLE
Selecting "Containers" in Remote Explorer doesn't finish loading targets

### ISSUE THREAD

**Issue by bamurtaugh** [microsoft/vscode-remote-release#6507]
**VS Code version:**
Version: 1.66.0-insider (user setup)
Commit: c63ed49b4b164210301f4f8a09079aa4de53b870
Date: 2022-03-25T14:30:54.397Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Windows_NT x64 10.0.22579

**Remote-Containers version:** v0.230.0 pre-release

Steps to Reproduce:

1. Open Remote Explorer
2. Target list loads when I select any option besides Containers from dropdown
3. Switch to Containers in the dropdown 
4. Containers list doesn't finish loading (Dev Volumes list does finish)
5. If I switch back to other remote options in dropdown, they load if I selected them in step 2, but don't load if I didn't click on them before the Containers hold up in step 3

I may have the most containers targets (although I do have quite a few WSL targets), so perhaps the quantity is causing the hang.

I wasn't sure which output logs to review, but in case it's helpful, here are the Extension Host logs:

```
[2022-03-25 12:21:57.287] [exthost] [info] Extension host with pid 27964 started
[2022-03-25 12:21:57.287] [exthost] [info] Skipping acquiring lock for c:\Users\brmurtau\AppData\Roaming\Code - Insiders\User\workspaceStorage\34db9ff7539d3a5e8dc26711d99aeb9f.
[2022-03-25 12:21:58.247] [exthost] [info] ExtensionService#_doActivateExtension vscode.microsoft-authentication, startup: false, activationEvent: 'onAuthenticationRequest:microsoft'
[2022-03-25 12:21:58.302] [exthost] [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: true, activationEvent: '*'
[2022-03-25 12:21:58.321] [exthost] [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
[2022-03-25 12:21:58.341] [exthost] [info] ExtensionService#_doActivateExtension vscode.ipynb, startup: true, activationEvent: '*'
[2022-03-25 12:21:58.358] [exthost] [info] ExtensionService#_doActivateExtension GitHub.codespaces, startup: true, activationEvent: '*'
[2022-03-25 12:22:00.370] [exthost] [info] ExtensionService#_doActivateExtension GitHub.vscode-pull-request-github, startup: true, activationEvent: '*'
[2022-03-25 12:22:00.683] [exthost] [info] ExtensionService#_doActivateExtension ms-vsliveshare.vsliveshare, startup: true, activationEvent: '*'
[2022-03-25 12:22:02.669] [exthost] [info] ExtensionService#_doActivateExtension vsls-contrib.codetour, startup: true, activationEvent: '*'
[2022-03-25 12:22:02.763] [exthost] [info] ExtensionService#_doActivateExtension vscode.npm, startup: true, activationEvent: 'workspaceContains:package.json'
[2022-03-25 12:22:02.785] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode-remote.remote-containers, startup: true, activationEvent: 'workspaceContains:.devcontainer/devcontainer.json'
[2022-03-25 12:22:05.440] [exthost] [info] Not activating extension 'ms-dotnettools.csharp': Timed out while searching for 'workspaceContains' pattern **/*.{csproj,sln,slnf,csx,cake}
[2022-03-25 12:22:05.559] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode-remote.remote-wsl, startup: false, activationEvent: 'onView:targetsWsl'
[2022-03-25 12:22:05.796] [exthost] [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
[2022-03-25 12:22:05.823] [exthost] [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
[2022-03-25 12:22:05.985] [exthost] [info] ExtensionService#_doActivateExtension vscode.github-authentication, startup: false, activationEvent: 'api', root cause: GitHub.codespaces
[2022-03-25 12:22:08.282] [exthost] [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onStartupFinished'
[2022-03-25 12:22:08.299] [exthost] [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
[2022-03-25 12:22:08.305] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode-remote.remote-wsl-recommender, startup: false, activationEvent: 'onStartupFinished'
[2022-03-25 12:22:08.318] [exthost] [info] ExtensionService#_doActivateExtension codespaces-contrib.codeswing, startup: false, activationEvent: 'onStartupFinished'
[2022-03-25 12:22:08.782] [exthost] [info] ExtensionService#_doActivateExtension dbaeumer.vscode-eslint, startup: false, activationEvent: 'onStartupFinished'
[2022-03-25 12:22:10.653] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode-remote.remote-ssh, startup: false, activationEvent: 'onView:sshHosts'
[2022-03-25 12:22:19.233] [exthost] [error] Activating extension ms-vsliveshare.vsliveshare failed due to an error:
[2022-03-25 12:22:19.233] [exthost] [error] Error: connect ENOENT \\.\pipe\vscode.d7c3fd912219a11369f5b3e9cd2b4d8a
    at PipeConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)
[2022-03-25 12:22:19.253] [exthost] [info] Eager extensions activated
[2022-03-25 12:22:48.226] [exthost] [info] ExtensionService#_doActivateExtension GitHub.remotehub-insiders, startup: false, activationEvent: 'onView:remoteHub.views.workspaceRepositories'
[2022-03-25 12:23:01.848] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode.remote-repositories, startup: false, activationEvent: 'onView:remoteHub.views.workspaceRepositories'
[2022-03-25 12:23:02.615] [exthost] [info] ExtensionService#_doActivateExtension GitHub.remotehub, startup: false, activationEvent: 'api', root cause: ms-vscode.remote-repositories
```

**Comment by chrmarti** [comment]
Could you also post `F1` > `Remote-Containers: Show Container Log`?

**Comment by bamurtaugh** [comment]
To clarify, I was experiencing this without having a dev container open, so I didn't have any Remote-Containers output logs.

If I try a slightly different scenario and open a dev container, I don't get any output if I open the Remote Explorer, then select `Remote-Containers: Show Container Log`:

![Recording 2022-03-29 at 10 38 13](https://user-images.githubusercontent.com/25310137/160672119-937989ec-dfa5-42fa-9c20-ea4b2be934cd.gif)
 
I reloaded the window, and now get logs from `Remote-Containers: Show Container Log` if I don't select the Remote Explorer:

```
[130 ms] Remote-Containers 0.230.0 in VS Code 1.66.0-insider (521a3d858570ebc9ce9d5602802f4525a02e045c).
[129 ms] Start: Resolving Remote
[170 ms] Start: Check Docker is running
[171 ms] Start: Run: docker version --format {{.Server.APIVersion}}
[854 ms] Server API version: 1.41
[857 ms] Start: Run: docker volume ls -q
[1427 ms] Start: Run: docker build -f C:\Users\brmurtau\AppData\Local\Temp\vsch\bootstrap-image\0.230.0\bootstrap.Dockerfile -t vsc-volume-bootstrap C:\Users\brmurtau\AppData\Local\Temp\vsch\bootstrap-image\0.230.0

[+] Building 0.5s (9/9) FINISHED
 => [internal] load build definition from bootstrap.Dockerfile             0.0s
 => => transferring dockerfile: 42B                                        0.0s
 => [internal] load .dockerignore                                          0.0s
 => => transferring context: 2B                                            0.0s
 => [internal] load metadata for mcr.microsoft.com/vscode/devcontainers/b  0.3s
 => [internal] load build context                                          0.0s
 => => transferring context: 5.90kB                                        0.0s
 => [1/4] FROM mcr.microsoft.com/vscode/devcontainers/base:0-alpine-3.14@  0.0s
 => CACHED [2/4] RUN apk add --no-cache  nodejs  npm  make  g++  docker-c  0.0s
 => CACHED [3/4] RUN cd && npm i node-pty                                  0.0s
 => CACHED [4/4] COPY .vscode-remote-containers /root/.vscode-remote-cont  0.0s
 => exporting to image                                                     0.0s
 => => exporting layers                                                    0.0s
 => => writing image sha256:5907ab4cdd15f5af2b44e299b1272051ac1477566985f  0.0s
 => => naming to docker.io/library/vsc-volume-bootstrap                    0.0s

Use 'docker scan' to run Snyk tests against images to find vulnerabilities and l
earn how to fix them
[3256 ms] Cloning Github repository: Microsoft/vscode-remote-try-python into /workspaces/vscode-remote-try-python

[3257 ms] Start: Run: docker run -d --mount type=volume,src=vscode-remote-try-python-41cc2ee6f8cb6b0cee4c350f9a78ef53,dst=/workspaces -v /var/run/docker.sock:/var/run/docker.sock vsc-volume-bootstrap sleep infinity
[5007 ms] Start: Run in container: /bin/sh
[5048 ms] Start: Launching Remote-Containers helper.
[5048 ms] ssh-agent: SSH_AUTH_SOCK in container (/tmp/vscode-ssh-auth-9dac7032128386423a686012e6c6404fb0df270d.sock) forwarded to local host (\\.\pipe\openssh-ssh-agent).
[5049 ms] Start: Run: gpgconf --list-dir agent-extra-socket
[5122 ms] findLocalWindowsExecutable: Exectuable 'gpgconf' not found on PATH 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Git LFS;C:\ProgramData\chocolatey\bin;C:\Program Files\Git\cmd;C:\Program Files\ffmpeg\bin;C:\Users\brmurtau\AppData\Local\Programs\Python\Python39;C:\Program Files\nodejs\;C:\Users\brmurtau\AppData\Local\Programs\Python\Python39\Scripts\;C:\Users\brmurtau\AppData\Local\Programs\Python\Python39\;;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\Microsoft VS Code Exploration\bin;C:\Users\brmurtau\AppData\Local\Programs\Python\Python39\Scripts\;C:\Users\brmurtau\AppData\Local\Programs\Python\Python39\;C:\Users\brmurtau\AppData\Local\Microsoft\WindowsApps;C:\Users\brmurtau\AppData\Local\GitHubDesktop\bin;C:\Users\brmurtau\AppData\Roaming\npm;C:\Users\brmurtau\AppData\Roaming\curl;;c:\Users\brmurtau\AppData\Roaming\Code - Insiders\User\globalStorage\ms-vscode-remote.remote-containers\cli-bin;C:\Users\brmurtau\AppData\Local\Microsoft\WindowsApps;C:\Users\brmurtau\AppData\Local\Programs\Microsoft VS Code Insiders\bin;C:\Users\brmurtau\AppData\Local\Programs\Microsoft VS Code\bin'.
[5130 ms] Start: Run in container: /bin/sh
[5170 ms] Start: Run in container: cat <<'EOF-/tmp/vscode-remote-containers-9dac7032128386423a686012e6c6404fb0df270d.js' >/tmp/vscode-remote-containers-9dac7032128386423a686012e6c6404fb0df270d.js
[6017 ms] 
[6017 ms] 
[6018 ms] Start: Run in container: cat <<'EOF-/tmp/vscode-remote-containers-server-9dac7032128386423a686012e6c6404fb0df270d.js' >/tmp/vscode-remote-containers-server-9dac7032128386423a686012e6c6404fb0df270d.js_1648575667883
[6034 ms] 
[6035 ms] 
[6049 ms] Start: Run in container: # Test for /root/.gitconfig and git
[6058 ms] 
[6059 ms] 
[6060 ms] Start: Run in container: # Copy C:\Users\brmurtau\.gitconfig to /root/.gitconfig
[6078 ms] 
[6079 ms] 
[6081 ms] Start: Run in container: command -v git >/dev/null 2>&1 && git config --global --replace-all credential.helper '!f() { node /tmp/vscode-remote-containers-9dac7032128386423a686012e6c6404fb0df270d.js $*; }; f' || true
[6095 ms] 
[6095 ms] 
[6099 ms] Start: Run in container: # Test for /root/.ssh/known_hosts and ssh
[6104 ms] 
[6104 ms] 
[6105 ms] Start: Run in container: # Copy C:\Users\brmurtau\.ssh\known_hosts to /root/.ssh/known_hosts
[6111 ms] 
[6111 ms] 
[6209 ms] Start: Run in container: cat /workspaces/vscode-remote-try-python/.devcontainer/devcontainer.json 2>/dev/null
[6216 ms] Start: Run in container: cat /workspaces/vscode-remote-try-python/.devcontainer/devcontainer.json 2>/dev/null
[7063 ms] Start: Run in container: docker volume create vscode
[9763 ms] Start: Run in container: docker ps -q -a --filter label=vsch.local.repository=https://github.com/Microsoft/vscode-remote-try-python --filter label=vsch.local.repository.volume=vscode-remote-try-python-41cc2ee6f8cb6b0cee4c350f9a78ef53 --filter label=vsch.local.repository.folder=vscode-remote-try-python --filter label=vsch.quality=insider
[10653 ms] Start: Run in container: docker inspect --type container 14c067e27bb0
[13530 ms] Start: Run in container: node /root/.vscode-remote-containers/dist/dev-containers-cli-0.230.0/dist/spec-node/devContainersSpecCLI.js up --container-data-folder .vscode-server-insiders/data/Machine --container-system-data-folder /var/vscode-server-insiders --workspace-folder /workspaces/vscode-remote-try-python --workspace-mount-consistency cached --id-label vsch.local.repository=https://github.com/Microsoft/vscode-remote-try-python --id-label vsch.local.repository.volume=vscode-remote-try-python-41cc2ee6f8cb6b0cee4c350f9a78ef53 --id-label vsch.local.repository.folder=vscode-remote-try-python --id-label vsch.quality=insider --log-level debug --config /workspaces/vscode-remote-try-python/.devcontainer/devcontainer.json --override-config /tmp/devcontainer-9ce440c7-c1ba-4e46-af8d-6d7aac4fca09.json --mount type=volume,source=vscode-remote-try-python-41cc2ee6f8cb6b0cee4c350f9a78ef53,target=/workspaces,external=true --mount type=volume,source=vscode,target=/vscode,external=true --skip-post-create --update-remote-user-uid-default off --mount-workspace-git-root true
[14387 ms] remote-containers 0.230.0.
[14386 ms] Start: Resolving Remote
[14405 ms] Start: Run: git rev-parse --show-cdup
[14420 ms] Start: Run: docker ps -q -a --filter label=vsch.local.repository=https://github.com/Microsoft/vscode-remote-try-python --filter label=vsch.local.repository.volume=vscode-remote-try-python-41cc2ee6f8cb6b0cee4c350f9a78ef53 --filter label=vsch.local.repository.folder=vscode-remote-try-python --filter label=vsch.quality=insider
[14460 ms] Start: Run: docker inspect --type container 14c067e27bb0
[14501 ms] Start: Inspecting container
[14501 ms] Start: Run: docker inspect --type container 14c067e27bb01b35536f5dc3bf585c905f0a211ba08749a12e1c7dccabd419ca
[14542 ms] Start: Run in container: /bin/sh
[14545 ms] Start: Run in container: uname -m
[14949 ms] x86_64
[14950 ms] 
[14950 ms] Start: Run in container: (cat /etc/os-release || cat /usr/lib/os-release) 2>/dev/null
[14959 ms] PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
[14959 ms] 
[14960 ms] Start: Run in container: cat /etc/passwd
[14968 ms] Start: Run in container: test -f '/var/vscode-server-insiders/.patchEtcEnvironmentMarker'
[14973 ms] 
[14974 ms] 
[14974 ms] Start: Run in container: test -f '/var/vscode-server-insiders/.patchEtcProfileMarker'
[14978 ms] 
[14978 ms] 
[15105 ms] Start: Run: docker inspect --type container 14c067e27bb01b35536f5dc3bf585c905f0a211ba08749a12e1c7dccabd419ca
[16753 ms] Start: Run in container: git rev-parse --show-cdup
[17389 ms] Start: Inspecting container
[17389 ms] Start: Run: docker inspect --type container 14c067e27bb01b35536f5dc3bf585c905f0a211ba08749a12e1c7dccabd419ca
[17927 ms] Start: Run in container: /bin/sh
[17958 ms] Start: Run in container: uname -m
[18778 ms] x86_64
[18779 ms] 
[18779 ms] Start: Run in container: (cat /etc/os-release || cat /usr/lib/os-release) 2>/dev/null
[18784 ms] PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
[18784 ms] 
[18785 ms] Start: Run in container: cat /etc/passwd
[18790 ms] Start: Setup shutdown monitor
[18792 ms] Forking shutdown monitor: c:\Users\brmurtau\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.230.0\dist\shutdown\shutdownMonitorProcess \\.\pipe\vscode-remote-containers-af29fd8a8aa0119111e254a2c4cf3d811a2c48f3-sock singleContainer Debug c:\Users\brmurtau\AppData\Roaming\Code - Insiders\logs\20220329T101258\exthost1\ms-vscode-remote.remote-containers 1648575661865
[18811 ms] Start: Run in container: test -d /home/vscode/.vscode-server-insiders
[18821 ms] 
[18822 ms] 
[18823 ms] Start: Run in container: test ! -f '/home/vscode/.vscode-server-insiders/data/Machine/.writeMachineSettingsMarker' && set -o noclobber && mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && { > '/home/vscode/.vscode-server-insiders/data/Machine/.writeMachineSettingsMarker' ; } 2> /dev/null
[18834 ms] 
[18834 ms] 
[18835 ms] Exit code 1
[18836 ms] Start: Run in container: cat /home/vscode/.vscode-server-insiders/data/Machine/settings.json
[18840 ms] {
        "terminal.integrated.profiles.linux": {
                "bash": {
                        "path": "/bin/bash"
                }
        },
        "python.defaultInterpreterPath": "/usr/local/bin/python",
        "python.languageServer": "Default",
        "python.linting.enabled": true,
        "python.linting.pylintEnabled": true,
        "python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
        "python.formatting.blackPath": "/usr/local/py-utils/bin/black",
        "python.formatting.yapfPath": "/usr/local/py-utils/bin/yapf",
        "python.linting.banditPath": "/usr/local/py-utils/bin/bandit",
        "python.linting.flake8Path": "/usr/local/py-utils/bin/flake8",
        "python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
        "python.linting.pycodestylePath": "/usr/local/py-utils/bin/pycodestyle",
        "python.linting.pydocstylePath": "/usr/local/py-utils/bin/pydocstyle",
        "python.linting.pylintPath": "/usr/local/py-utils/bin/pylint",
        "remote.portsAttributes": {
                "9000": {
                        "label": "Hello Remote World",
                        "onAutoForward": "notify"
                }
        }
}
[18841 ms] 
[18842 ms] Start: Run in container: test -d /home/vscode/.vscode-server-insiders/bin/521a3d858570ebc9ce9d5602802f4525a02e045c-insider
[18846 ms] 
[18846 ms] 
[18846 ms] Start: Run in container: test -d /vscode/vscode-server-insiders/bin/linux-x64/521a3d858570ebc9ce9d5602802f4525a02e045c-insider
[18850 ms] 
[18851 ms] 
[18851 ms] Start: Run in container: /bin/sh
[18851 ms] Start: Launching Remote-Containers helper.
[18852 ms] ssh-agent: SSH_AUTH_SOCK in container (/tmp/vscode-ssh-auth-826b82ce2eb72f499b710c14247276839e016530.sock) forwarded to local host (\\.\pipe\openssh-ssh-agent).
[18852 ms] Start: Run: gpgconf --list-dir agent-extra-socket
[18877 ms] Start: Run in container: touch '/vscode/vscode-server-insiders/bin/linux-x64/521a3d858570ebc9ce9d5602802f4525a02e045c-insider'
[18901 ms] findLocalWindowsExecutable: Exectuable 'gpgconf' not found on PATH 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Git LFS;C:\ProgramData\chocolatey\bin;C:\Program Files\Git\cmd;C:\Program Files\ffmpeg\bin;C:\Users\brmurtau\AppData\Local\Programs\Python\Python39;C:\Program Files\nodejs\;C:\Users\brmurtau\AppData\Local\Programs\Python\Python39\Scripts\;C:\Users\brmurtau\AppData\Local\Programs\Python\Python39\;;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\Microsoft VS Code Exploration\bin;C:\Users\brmurtau\AppData\Local\Programs\Python\Python39\Scripts\;C:\Users\brmurtau\AppData\Local\Programs\Python\Python39\;C:\Users\brmurtau\AppData\Local\Microsoft\WindowsApps;C:\Users\brmurtau\AppData\Local\GitHubDesktop\bin;C:\Users\brmurtau\AppData\Roaming\npm;C:\Users\brmurtau\AppData\Roaming\curl;;c:\Users\brmurtau\AppData\Roaming\Code - Insiders\User\globalStorage\ms-vscode-remote.remote-containers\cli-bin;C:\Users\brmurtau\AppData\Local\Microsoft\WindowsApps;C:\Users\brmurtau\AppData\Local\Programs\Microsoft VS Code Insiders\bin;C:\Users\brmurtau\AppData\Local\Programs\Microsoft VS Code\bin'.
[18907 ms] Start: Run in container: /bin/sh
[18908 ms] userEnvProbe: loginInteractiveShell (default)
[18909 ms] userEnvProbe shell: /bin/bash
[18911 ms] Start: Run in container: # Test for /home/vscode/.ssh/known_hosts and ssh
[18918 ms] /home/vscode/.ssh/known_hosts exists
[18918 ms] 
[18918 ms] Exit code 1
[18943 ms] Start: Run in container: cat <<'EOF-/tmp/vscode-remote-containers-826b82ce2eb72f499b710c14247276839e016530.js' >/tmp/vscode-remote-containers-826b82ce2eb72f499b710c14247276839e016530.js
[19580 ms] 
[19581 ms] 
[19581 ms] Start: Run in container: command -v git >/dev/null 2>&1 && git config --system --replace-all credential.helper '!f() { /home/vscode/.vscode-server-insiders/bin/521a3d858570ebc9ce9d5602802f4525a02e045c-insider/node /tmp/vscode-remote-containers-826b82ce2eb72f499b710c14247276839e016530.js $*; }; f' || true
[19592 ms] 
[19592 ms] 
[19594 ms] Start: Run in container: for pid in `cd /proc && ls -d [0-9]*`; do { echo $pid ; readlink /proc/$pid/cwd ; readlink /proc/$pid/ns/mnt ; cat /proc/$pid/stat | tr "
[19770 ms] 
[19771 ms] 
[19771 ms] Start: Run in container: cat <<'EOF-/tmp/vscode-remote-containers-server-826b82ce2eb72f499b710c14247276839e016530.js' >/tmp/vscode-remote-containers-server-826b82ce2eb72f499b710c14247276839e016530.js_1648575681636
[19781 ms] 
[19781 ms] 
[19943 ms] Extension host agent is already running.
[19943 ms] Start: Run in container: cat '/home/vscode/.vscode-server-insiders/data/Machine/.devport-521a3d858570ebc9ce9d5602802f4525a02e045c-insider' 2>/dev/null
[19947 ms] 46175
[19947 ms] 
[19948 ms] Start: Run in container: cat '/home/vscode/.vscode-server-insiders/data/Machine/.connection-token-521a3d858570ebc9ce9d5602802f4525a02e045c-insider'
[19952 ms] 42c42998-ee4d-4d13-bd5d-17a8c1e49f79
[19952 ms] 
[19952 ms] Port forwarding for container port 46175 starts listening on local port.
[19954 ms] Port forwarding local port 57991 to container port 46175
[19954 ms] Start: Run in container: node /root/.vscode-remote-containers/dist/dev-containers-cli-0.230.0/dist/spec-node/devContainersSpecCLI.js run-user-commands --container-data-folder .vscode-server-insiders/data/Machine --container-system-data-folder /var/vscode-server-insiders --workspace-folder /workspaces/vscode-remote-try-python --id-label vsch.local.repository=https://github.com/Microsoft/vscode-remote-try-python --id-label vsch.local.repository.volume=vscode-remote-try-python-41cc2ee6f8cb6b0cee4c350f9a78ef53 --id-label vsch.local.repository.folder=vscode-remote-try-python --id-label vsch.quality=insider --log-level debug --config /workspaces/vscode-remote-try-python/.devcontainer/devcontainer.json --override-config /tmp/devcontainer-9ce440c7-c1ba-4e46-af8d-6d7aac4fca09.json --default-user-env-probe loginInteractiveShell --skip-non-blocking-commands false --prebuild false --stop-for-personalization true --remote-env SSH_AUTH_SOCK=/tmp/vscode-ssh-auth-826b82ce2eb72f499b710c14247276839e016530.sock --remote-env REMOTE_CONTAINERS_IPC=/tmp/vscode-remote-containers-ipc-826b82ce2eb72f499b710c14247276839e016530.sock --remote-env REMOTE_CONTAINERS=true --mount-workspace-git-root true
[20018 ms] userEnvProbe PATHs:
Probe:     '/usr/local/pip-global:/usr/local/share/nvm/current/bin:/usr/local/bin:/usr/local/pip-global/bin:/usr/local/share/nvm/versions/node/v16.14.0/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/py-utils/bin:/home/vscode/.local/bin:/usr/local/py-utils/bin'
Container: '/usr/local/pip-global/bin:/usr/local/share/nvm/current/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/py-utils/bin'
[20031 ms] Port forwarding connection from 57992 > 57991 > 46175 in the container.
[20032 ms] Start: Run in container: /home/vscode/.vscode-server-insiders/bin/521a3d858570ebc9ce9d5602802f4525a02e045c-insider/node -e 
[20982 ms] remote-containers 0.230.0.
[20981 ms] Start: Run: git rev-parse --show-cdup
[20992 ms] Port forwarding 57992 > 57991 > 46175 stderr: Connection established
[21001 ms] Start: Run: docker ps -q -a --filter label=vsch.local.repository=https://github.com/Microsoft/vscode-remote-try-python --filter label=vsch.local.repository.volume=vscode-remote-try-python-41cc2ee6f8cb6b0cee4c350f9a78ef53 --filter label=vsch.local.repository.folder=vscode-remote-try-python --filter label=vsch.quality=insider
[21011 ms] Port forwarding connection from 57996 > 57991 > 46175 in the container.
[21011 ms] Start: Run in container: /home/vscode/.vscode-server-insiders/bin/521a3d858570ebc9ce9d5602802f4525a02e045c-insider/node -e 
[21082 ms] Start: Run: docker inspect --type container 14c067e27bb0
[21192 ms] Start: Inspecting container
[21192 ms] Start: Run: docker inspect --type container 14c067e27bb01b35536f5dc3bf585c905f0a211ba08749a12e1c7dccabd419ca
[21274 ms] Start: Run in container: /bin/sh
[21281 ms] Start: Run in container: uname -m
[21820 ms] x86_64
[21820 ms] 
[21820 ms] Start: Run in container: (cat /etc/os-release || cat /usr/lib/os-release) 2>/dev/null
[21826 ms] PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
[21826 ms] 
[21827 ms] Start: Run in container: cat /etc/passwd
[21835 ms] userEnvProbe: loginInteractiveShell (default)
[21836 ms] Start: Run in container: mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && CONTENT="$(cat '/home/vscode/.vscode-server-insiders/data/Machine/.onCreateCommandMarker' 2>/dev/null || echo ENOENT)" && [ "${CONTENT:-2022-03-29T17:35:00.877217546Z}" != '2022-03-29T17:35:00.877217546Z' ] && echo '2022-03-29T17:35:00.877217546Z' > '/home/vscode/.vscode-server-insiders/data/Machine/.onCreateCommandMarker'
[21836 ms] userEnvProbe shell: /bin/bash
[21837 ms] Start: Run in container: /bin/bash -lic echo -n 172ef36662920607b2cf2afc07cc536e; cat /proc/self/environ; echo -n 172ef36662920607b2cf2afc07cc536e
[21847 ms] 
[21847 ms] 
[21847 ms] Exit code 1
[21848 ms] Start: Run in container: mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && CONTENT="$(cat '/home/vscode/.vscode-server-insiders/data/Machine/.updateContentCommandMarker' 2>/dev/null || echo ENOENT)" && [ "${CONTENT:-2022-03-29T17:35:00.877217546Z}" != '2022-03-29T17:35:00.877217546Z' ] && echo '2022-03-29T17:35:00.877217546Z' > '/home/vscode/.vscode-server-insiders/data/Machine/.updateContentCommandMarker'
[21858 ms] 
[21859 ms] 
[21859 ms] Exit code 1
[21859 ms] Start: Run in container: mkdir -p '/home/vscode/.vscode-server-insiders/data/Machine' && CONTENT="$(cat '/home/vscode/.vscode-server-insiders/data/Machine/.postCreateCommandMarker' 2>/dev/null || echo ENOENT)" && [ "${CONTENT:-2022-03-29T17:35:00.877217546Z}" != '2022-03-29T17:35:00.877217546Z' ] && echo '2022-03-29T17:35:00.877217546Z' > '/home/vscode/.vscode-server-insiders/data/Machine/.postCreateCommandMarker'
[21869 ms] 
[21869 ms] 
[21869 ms] Exit code 1
[21976 ms] Start: Run in container: # Test for /home/vscode/.gitconfig and git
[21980 ms] /home/vscode/.gitconfig exists
[21980 ms] 
[21980 ms] Exit code 1
[21981 ms] Start: Run in container: command -v git >/dev/null 2>&1 && git config --global --replace-all credential.helper '!f() { /home/vscode/.vscode-server-insiders/bin/521a3d858570ebc9ce9d5602802f4525a02e045c-insider/node /tmp/vscode-remote-containers-826b82ce2eb72f499b710c14247276839e016530.js $*; }; f' || true
[21987 ms] 
[21988 ms] 
[21988 ms] Start: Run: docker rm -f 98d06f75120efa7364db966a3fb3c3821aa80569561bd933fdbaabdccec02a4f
[22324 ms] Port forwarding 57996 > 57991 > 46175 stderr: Connection established
[23882 ms] Container server terminated (code: 137, signal: null).
[24865 ms] Start: Run in container: cat /proc/932/environ
[24866 ms] Stdin closed!
[24866 ms] Start: Run in container: cat /proc/932/environ
[24894 ms] Start: Run in container: rm '/tmp/vscode-remote-containers-ipc-033afe0b100699f5cd4bb2870bce01c5feb83759.sock' && ln -s '/tmp/vscode-remote-containers-ipc-826b82ce2eb72f499b710c14247276839e016530.sock' '/tmp/vscode-remote-containers-ipc-033afe0b100699f5cd4bb2870bce01c5feb83759.sock'
[24914 ms] 
[24915 ms] 
[24915 ms] Start: Run in container: rm '/tmp/vscode-ssh-auth-033afe0b100699f5cd4bb2870bce01c5feb83759.sock' && ln -s '/tmp/vscode-ssh-auth-826b82ce2eb72f499b710c14247276839e016530.sock' '/tmp/vscode-ssh-auth-033afe0b100699f5cd4bb2870bce01c5feb83759.sock'
[24965 ms] 
[24965 ms] 
```

But if I open the Remote Explorer again, it hangs on loading Containers, and `Remote-Containers: Show Container Log` does nothing (same behavior as gif above).

**Comment by chrmarti** [comment]
Not sure what might be going on. The container log would show any commands run also when not connected to a container, but this sounds like it doesn't get there.

Could you check if it works with 0.224.3?

**Comment by bamurtaugh** [comment]
> Could you check if it works with 0.224.3?

I just manually switched to 0.224.3 through command `Extensions: Install Specific Version of Extension...`. 

The Remote Explorer view with Containers selected in the dropdown list did load!

I updated the extension to the latest stable (v0.231.1), and the Containers list populates there too. I finally switched back to the latest pre-release (v0.232.1), and now it seems to work 🤔. I'm not sure if getting it to populate once through 0.224.3 helped and the list can pull from a cache, or it was a fleeting issue for a few days.
